### ISSUE TITLE
Cannot find rule for template-accessibility-tabindex-no-positive

### ISSUE THREAD

**Issue by JamesIves** [angular-eslint/angular-eslint#324]
I'm running into an issue when adding `@angular-eslint/template/accessibility-tabindex-no-positive` into my template eslint rules. I end up with the error saying it cannot find the rule for it. All of the other accessibility templates work but for some reason this is throwing an error.  I'm running version 1.1.0, happy to provide any additional details if need be.

Also just wanted to say this package is fantastic, and a real lifesaver.

**Comment by JamesIves** [comment]
Had a cheeky look through the code, it looks like this is being exported as `@angular-eslint/template/no-positive-tabindex`. While this works the docs are a little bit confusing.

**Comment by JamesHenry** [comment]
Thanks @JamesIves - I have now updated the rules list table generation to include some more information, including the full name of the ESLint equivalent for each Codelyzer rule.

Cheers!

**Comment by rafaelss95** [comment]
@JamesHenry Taking the fact we are talking about accessibility, is there any reason for dropping `accessibility` from this rule? This is also related to #238.

**Comment by JamesIves** [comment]
> Thanks @JamesIves - I have now updated the rules list table generation to include some more information, including the full name of the ESLint equivalent for each Codelyzer rule. You can see it in the updated repo README.
> 
> Cheers!

Thanks! This is super helpful.
